roles/openshift_cluster_monitoring_operator/files/cluster-monitoring-operator.yaml

```diff
@@ -38,104 +38,92 @@ objects:
   metadata: {name: cluster-monitoring-operator}
   rules:
   - apiGroups: [rbac.authorization.k8s.io]
-    resources: [roles, rolebindings, clusterroles, clusterrolebindings]
```

Comment:
What's the change here? It just rearranges the order of resources  - does it matter?

Reply:
This is copied verbatim from our upstream source [0]. In short, yes, it does matter that these are sorted. This resource is a concatenation of several roles and before performing a sorting of all the rules we had duplicate rules in roles and generated a non-stable output manifest. [1]

[0] https://github.com/openshift/cluster-monitoring-operator/blob/master/manifests/cluster-monitoring-operator-role.yaml
[1] https://github.com/openshift/cluster-monitoring-operator/issues/59